Cargo.toml

```diff
@@ -93,6 +93,7 @@ base64 = "0.10"
 safemem = "0.3"
 cop_datatype = { path = "components/cop_datatype" }
 smallvec = { version = "0.6", features = ["union"] }
+procinfo = { git = "https://github.com/ice1000/procinfo-rs", rev = "6d97b3ebd78b7d7ce19c6bf83b4463e3a6620dd2" }
```

Comment:
@siddontang I think this repo should be transferred to the tikv org.
The original version is not maintained and the original author disappears from GitHub.